src/Http/TreeRouteStack.php

```diff
@@ -262,9 +261,9 @@ public function addPrototypes($routes)
     /**
      * Add a prototype.
      *
-     * @param  string $name
-     * @param  mixed  $route
-     * @return TreeRouteStack
+     * @param string                          $name
+     * @param string|array|Traversable|TRoute $route
```

Comment:
`array|Traversable` to `iterable` as above

Reply:
58aa9bfcde673252fe98629c3067838ffe1ba62b